tsctl

```diff
@@ -364,6 +365,51 @@ class CreateTimelineFromCsv(CreateTimelineBase):
         self.create_searchindex(timeline_name, index_name)
 
 
+class PurgeTimeline(Command):
+    """Delete timeline permanently from Timesketch and Elasticsearch."""
+    option_list = (
+        Option(u'--index', u'-i', dest=u'index_name', required=True),
+    )
+
+    def __init__(self):
+        super(PurgeTimeline, self).__init__()
+
+    def run(self, index_name):
+        """Delete timeline in both Timesketch snd Elasticsearch.
+
+        Args:
+            index_name: The name of the index in Elasticsearch
+        """
+        index_name = unicode(index_name.decode(encoding=u'utf-8'))
+        searchindex = SearchIndex.query.filter_by(index_name=index_name).first()
+
+        if not searchindex:
+            sys.stdout.write(u'No such index\n')
+            sys.exit()
+
+        es = ElasticsearchDataStore(
+            host=current_app.config[u'ELASTIC_HOST'],
+            port=current_app.config[u'ELASTIC_PORT'])
+
+        timelines = Timeline.query.filter_by(searchindex=searchindex).all()
+        sketches = [t.sketch for t in timelines if
+                    t.sketch and t.sketch.get_status.status != u'deleted']
+        if sketches:
+            sys.stdout.write(u'WARNING: This timeline is in use by:\n')
+            for sketch in sketches:
+                sys.stdout.write(u' * {0:s}\n'.format(sketch.name))
```

Comment:
This is just a habit of mine when writing CLI tools. I want to control where to put the output (stdout, stderr etc).